install/kubernetes/helm/istio/values-istio-sds-auth.yaml

```diff
@@ -11,10 +11,12 @@ global:
     udsPath: "unix:/var/run/sds/uds_path"
     useNormalJwt: true
 
+  useMCP: true
```

Comment:
Can you check why you have to redefine this? I think it should be true by default.

Reply:
removed here (I originally copied change from istio/test-values/values-istio-auth-sds.yaml which useMCP is false by default).